engines/twp/actorlib.cpp

```diff
@@ -754,8 +754,8 @@ static SQInteger actorWalkTo(HSQUIRRELVM v) {
 		if (SQ_FAILED(sqget(v, 4, y)))
 			return sq_throwerror(v, "failed to get y");
 		Facing *facing = nullptr;
+		int dir;
```

Comment:
This is a particularly critical one, since otherwise the facing point is pointing at a local variable i.e. dir which has gone out of scope and thus may not exist. This might work, but is likely to crash / segfault or do odd things.